main.go

```diff
@@ -81,47 +83,58 @@ func main() {
 		HealthProbeBindAddress: config.ProbeAddr,
 		LeaderElection:         config.EnableLeaderElection,
 		LeaderElectionID:       "06d035fb.mongodb.com",
-		Namespace:              config.WatchedNamespaces,
+		Namespace:              "",
```

Comment:
I'm not particularly sure why the Manager has the ability to just watch one namespace, but from what I've seen `controller-runtime` prefers using predicates to filter and select namespaces for watching.

Reply:
"" causes it to watch all namespaces, and then the Predicates filter the namespaces.